### ISSUE TITLE
Cowin site and API says slots available but on logging in, no available slots

### ISSUE THREAD

**Issue by thesuhas** [cowinapi/developer.cowin#260]
Hey, so the request from the API and the Cowin Homepage says a slot is available (Typically Apollo Hospitals, Jayanagar) but on logging in, it shows that it is fully booked. This is not getting updated on the homepage and as a result API calls over a period of time return free slots even though there are none.

![image](https://user-images.githubusercontent.com/54014218/118583428-ae398900-b7b2-11eb-947e-37aae791939d.png)
The above image is on the Cowin Homepage.

![image](https://user-images.githubusercontent.com/54014218/118583579-eb9e1680-b7b2-11eb-94a0-1281b26e6a25.png)
The above image is on logging in. 

There seems to be some sort of inconsistencies between the API/Cowin Slots and on authentication

**Edit:** There seems to be a temporary fix by using `available_capacity_dose1` instead of `available_capacity` as `available_capacity_dose1` is what is actually being used for booking and `available_capacity` is just being used to display numbers in the Cowin UI on the homepage.
In addition, looks like `available_capacity` is not always being updated and is not always a sum of `dose1` and `dose2` fields.

More info available in Issue #244

**Comment by xsreality** [comment]
That's right. When not logged in, the UI shows `available_capacity` which is supposed to be sum of dose1 and dose2 (though in some cases it is not which is likely a bug). 

When logged in, portal knows which dose (first or second) the beneficiary is looking for and will show that number instead of `available_capacity`.

It makes sense but is still confusing. They can improve on this by showing both dose1 and dose2 numbers separately per session when not logged in so that people can decide to login and book based on which dose they are looking for.
